source/Nabla/CIrrDeviceWin32.h

```diff
@@ -99,7 +99,7 @@ namespace nbl
             }
 
             //! switchs to fullscreen
-            bool switchToFullScreen(bool reset=false);
+            bool switchToFullscreen(bool reset=false) override;

```

Comment:
that should not be in device... but in the Window class

Reply:
dont review this, im just trying to run egl on current API. I want to have it tested before I turn whole engine upside down